test/factoryTest.js

```diff
@@ -1,64 +1,75 @@
-const SortitionPoolFactory = artifacts.require(
-  "./contracts/SortitionPoolFactory.sol",
-)
-const SortitionPool = artifacts.require("./contracts/SortitionPool.sol")
-const StakingContractStub = artifacts.require("StakingContractStub.sol")
+const { helpers, ethers } = require("hardhat")
+const { expect } = require("chai")
+const utils = require("./utils")
 
-const { mineBlocks } = require("./mineBlocks")
```

Comment:
We still have `test/mineBlocks.js`. Is it needed?

Reply:
Done in https://github.com/keep-network/sortition-pools/pull/127/commits/1fdcda0aae9ee9065e524e6280f2dc2157e2aaf3.